vulnerable_people_form/templates/not-eligible-medical.html

```diff
@@ -7,7 +7,12 @@
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-8">
     {{ title_text }}
   </h1>
-  <p class="govuk-body">This service is for people who are classed as clinically extremely vulnerable to coronavirus.</p>
+  <p class="govuk-body">Based on what you’ve told us, you’re not eligible for help through this service</p>
+  <p class="govuk-body">This is because:</p>
+  <ul class="govuk-list">
+    <li>- the NHS or your doctor has not told you that you’re clinically extremely vulnerable to coronavirus</li>
```

Comment:
The list items are not showing as bullet points. Could it be because of the `-` at the start?

Reply:
Or it may need the "govuk-list--bullet" class in the `ul` element

Reply:
the items are not meant to show as bullet points

Reply:
https://www.figma.com/file/yqy8aBWZsu5LuYnmIaGTj7/Extremely-vulnerable-people-form?node-id=5197%3A0